scripts/import_docs.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
```

Comment:
Remember that a script in an open source project should always contain the title and the license at the top.